docs/assessment.md

```diff
@@ -0,0 +1,188 @@
+# UCX Assessment Introduction
+This document describes the Assessment Report generated from the UCX tools. The main assessment report includes dashlets, widgets and details of the assessment findings and common recommendations made based on the Assessment Finding (AF) Index entry.
+
+# Assessment Report Summary
+The Assessment Report (Main) is the output of the Databricks Labs UCX assessment workflow. This report queries the $inventory database (e.g. `ucx`) and summarizes the findings of the assessment. The link to the Assessment Report (Main) can be found in your home folder, under `.ucx` in the README.py file. The user may also directly navigate to the Assessment report by clicking on `Dashboards` icon on the left to find the Dashboard.
+
+# Assessment Dashlets
+<img width="1655" alt="image" src="https://github.com/databrickslabs/ucx/assets/1122251/808f7c68-fcc7-4caa-bab2-03f49a382256">
```

Comment:
Images are not version controllable, try to avoid them. We're changing to lakeview soon, anyway

Reply:
Please take a look at the doc, the images really pull it together for the reader, breaking the text into bite size bits.

As far as versioning images go, the images are immutable and not intended to be versioned in place, github does a really nice job. When Lakeview comes along it's just 5 copy paste operations.

Reply:
![image](https://github.com/databrickslabs/ucx/assets/259697/809509d7-5f2e-47c0-a8f6-eef50da934b3)

Here's what I'm talking about.

Besides, we're changing the dashboards to lakeview soon